frontend/src/pages/FileUploadPage/components/FileDropzone.tsx

```diff
@@ -1,26 +1,70 @@
 /* eslint-disable react/jsx-props-no-spreading */
-import React, { ReactElement } from "react";
-import Dropzone from "react-dropzone";
+import React, { ReactElement, useCallback, useState } from "react";
+import { useDropzone, FileWithPath } from "react-dropzone";
 import styles from "../FileUpload.module.css";
 import FileDropZoneControls from "./FileDropzoneControls";
 
 function FileDropZone(): ReactElement {
+  const [files, setFiles] = useState<FileWithPath[]>([]);
+
+  const pstyle = { fontWeight: "bold", fontSize: "22px" }; // add to style sheet
+
+  const clearAcceptedFiles = () => {
+    setFiles([]);
+  };
+
+  const onDrop = useCallback((acceptedFiles: FileWithPath[]) => {
+    console.log(acceptedFiles);
+
+    if (acceptedFiles?.length) {
+      setFiles((previousFiles) => [...previousFiles, ...acceptedFiles]);
+    }
+  }, []);
+
+  const { getRootProps, open, getInputProps, isDragActive } = useDropzone({
+    onDrop,
+    accept: {
+      "application/json": [".json"],
+      "application/xml": [".xml"],
+    },
+  });
+
+  const acceptedFileItems = files.map((file: FileWithPath) => (
+    <li key={file.path}>
+      {file.path} - {file.size} bytes
+    </li>
+  ));
+
   return (
-    <div className={styles.main}>
-      <Dropzone>
-        {({ getRootProps, getInputProps }) => (
+    <>
+      <FileDropZoneControls onRadioChange={clearAcceptedFiles} />
```

Comment:
I think this is a good pattern to communicate with subcomponents that we will use for the rest of the project